torch/fx/passes/utils/matcher_utils.py

```diff
@@ -106,8 +106,20 @@ def _match_attributes(self, pn: Node, gn: Node) -> bool:
         # Attributes matching is complicated. Right now we only support matching constant tensor
         assert isinstance(pn.target, str), f"pn.target {pn.target} must be a string."
         assert isinstance(gn.target, str), f"gn.target {gn.target} must be a string."
-        pn_value = getattr(pn.graph.owning_module, pn.target)
-        gn_value = getattr(gn.graph.owning_module, gn.target)
+
+        # TODO(tmanlaibaatar) should probably make this actual API
+        def _getattr(model: torch.fx.GraphModule, attr_name: str):
```

Comment:
Maybe add comment about why we're doing this and a unit test to ensure it functions properly